
[Executed at: Wed Mar 14 1:39:15 PDT 2018]

Traceback (most recent call last):
  File "/home/ccc_v1_c79523__48809/asn24778_45/asn24782_4/asnlib/submission.py", line 8, in <module>
    from q6_work import find_path
  File "./q6_work.py", line 27, in <module>
    print(get_stationline(mrt_d))
  File "./q6_work.py", line 15, in get_stationline
    for i in (read_stations.keys()):
AttributeError: 'function' object has no attribute 'keys'
Command exited with non-zero status 1
